packages/starrating/src/css/index.js

```diff
@@ -49,27 +49,32 @@ export default {
     ...resetButton,
     lineHeight: 0
   },
-  [`${BASE_CLASSNAME}__star--active`]: {
-    color: core.colors.yellow
-  },
-  [`${BASE_CLASSNAME}__star--interactive`]: {
-    cursor: 'pointer'
-  },
+
   [`${BASE_CLASSNAME}__star--theme-${themeNames.dark}`]: {
-    color: core.colors.gray03
+    color: core.colors.gray04
   },
   [`${BASE_CLASSNAME}__star--theme-${themeNames.light}`]: {
     color: core.colors.gray01
   },
 
   [`${BASE_CLASSNAME}__star__half__secondary`]: {},
   [`${BASE_CLASSNAME}__star__half__secondary--theme-${themeNames.dark}`]: {
-    color: core.colors.gray03
+    color: core.colors.gray04
   },
   [`${BASE_CLASSNAME}__star__half__secondary--theme-${themeNames.light}`]: {
     color: core.colors.gray01
   },
 
+  [`${BASE_CLASSNAME}__star--active`]: {
+    color: core.colors.yellow
+  },
+  [`${BASE_CLASSNAME}__star--bright`]: {
```

Comment:
"Bright" may very well be the best name for this, I'm having a thought that might be something good to consider: traditionally we've tried to default these modifier names to match the props as closely as possible.  Two basic reasons I feel have been worthwhile: Keep from introducing new variable names for things (just call it the same idea throughout).  The other is to try to make the style rules more apparent in the css itself rather than having to look through the component internals.  

In this case, to get there, we could create a selector like:

```
.psds-starrating__star--interactive.psds-starrating__star--not-value {
 ...
}
```

It's not as short as "bright" but it doesn't intro new names and describes what state it's styling in the selector.

Anyway, something to consider.

Reply:
If the `bright` prop was removed, this state could be calculated in the `Star` itself so you didn't have to pass the prop either.

Reply:
yeah this is a tricky one. there are a lot of variations and i'm trying to keep the individual Star component as simple as possible while doing all the logic in the StarRating. i don't have the perfect answer but i think keeping these props as presentational give a good about of flexibility...let me noodle on it

Reply:
Yes, straight-forward Star component also seems desirable.  Might be a competing priority.  I'm not sure which one has higher priority.